web/controllers/v1/password_reset_controller.ex

```diff
@@ -9,6 +9,9 @@ defmodule Cog.V1.PasswordResetController do
          {:ok, _} <- Users.request_password_reset(user) do
       send_resp(conn, :no_content, "")
     else
+      {:error, :not_found} ->
+        Logger.warn("Unknown email address sent for password reset")
+        send_resp(conn, :no_content, "")
```

Comment:
It would. I'm sending the same response as success in this case though so nefarious people don't just spam the endpoint to try and guess users emails. Maybe it doesn't matter though?